### ISSUE TITLE
Metadata about CIDR's or IPs

### ISSUE THREAD

**Issue by ekarlso** [metal-stack/go-ipam#99]
Is there a idiomatic way to store metadata like name and alike for the various CIDRs/ IPs ?

**Comment by majst01** [comment]
No, this is not implemented and also not desired, because go-ipam is designed to be as fast as possible, and adding meta data to the IPs/Prefixes would add quite a lot of data.

We at metal-stack store these information alongside the gathered IP/Prefix.
